en/docs/api-analytics/samples/publishing-analytics-events-to-external-systems.md

```diff
@@ -0,0 +1,257 @@
+
+    logger.reporter.name = <PackageName>
+    logger.reporter.level = INFO
+
+## Visualizing Logs
+
+WSO2 API-M logs are structured in a way that we can easily plug them into a log visualization tool in order to visualize them. As a result, this gives the capability to monitor analytics traffic for a particular API. For this purpose, ELK is selected.
+
+### Configuring ELK
+
+#### Installing Elasticsearch
+
+[Install Elasticsearch](https://www.elastic.co/guide/en/elastic-stack-get-started/7.13/get-started-elastic-stack.html#install-elasticsearch) according to your operating system.
+
+Make sure Elasticsearch is [up and running](https://www.elastic.co/guide/en/elastic-stack-get-started/7.13/get-started-elastic-stack.html#_make_sure_elasticsearch_is_up_and_running).
+
+#### Installing Kibana
+
+[Install Kibana](https://www.elastic.co/guide/en/elastic-stack-get-started/7.13/get-started-elastic-stack.html#install-kibana) according to your operating system.
+
+[Launch](https://www.elastic.co/guide/en/elastic-stack-get-started/7.13/get-started-elastic-stack.html#_launch_the_kibana_web_interface) the Kibana web interface.
+
+#### Installing Filebeat
+
+[Install Filebeat](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#installation) according to your operating system.
+
+[Connect](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#set-connection) to Elastic Stack.
+
+#### Collecting Log Data
+
+Add bellow configurations to feed WSO2 API-M logs in to Filebeat,
+
+Replace `<API-M HOME>` with the location of your `API-M Home` directory.
+
+    filebeat.inputs:
+    - type: log
+    enabled: true
+    paths:
+        - /<API-M HOME>/repository/logs/wso2carbon.log
+
+[Set up](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#setup-assets) assets
+
+In case of a failure with the above command, use below to set up assets,
+
+    filebeat -e
+
+[Start](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#start) Filebeat
+
+### View Data on Kibana
+
+Filebeat comes with pre-built Kibana dashboards and UIs for visualizing log data.
+
+[Launch Kibana and discover](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#view-data) log data.
+
+Once you have followed and completed the above steps successfully, you will be able to visualize log data as shown below,
+
+![Logs listed in kibana]({{base_path}}/assets/img/analytics/samples/logs_listed_in_kibana.png)
+
+#### Filtering Total Analytics Traffic
+
+It is possible to view the analytics traffic by applying a filter as shown below,
+
+Replace `<MetricReporterImplClass>` with the class name given to the MetricReporter implementation class that you have created with your sample.
+
+![Total analytics traffic filter]({{base_path}}/assets/img/analytics/samples/total_analytics_traffic_filter.png)
```

Comment:
All images should be zoomable.
Do not add _ in image names. Instead replace it with - a hyphen.